src/Workspaces/Core/Portable/Execution/Serializer.cs

```diff
@@ -199,7 +199,8 @@ private string GetLanguageName(object value)
                 IOptionsSerializationService service;
                 if (_lazyLanguageSerializationService.TryGetValue(languageName, out service))
```

Comment:
_lazyLanguageService acts as a cache so that I don't do code below multiple times. for language which doesn't support the service, it will have null as value.